src/ribasim_nl/ribasim_nl/raster.py

```diff
@@ -0,0 +1,178 @@
+# %%
```

Comment:
Maybe better if you remove these all at once rather than me making review suggestions.